language-support/hs/bindings/test/DA/Ledger/Sandbox.hs

```diff
@@ -77,6 +81,7 @@ getListeningLine :: Handle -> IO String
 getListeningLine h = loop where
     loop = do
         line <- hGetLine h
+        print line
```

Comment:
Debugging?

Reply:
oops that what I get for trying to fix bugs quickly